src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/maui.aotprofile.txt

```diff
@@ -1836,20 +1457,14 @@ Methods:
-	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Internals.TemplatedItemsList`2<Microsoft.Maui.Controls.MultiPage`1<Microsoft.Maui.Controls.Page>, Microsoft.Maui.Controls.Page>:get_GroupHeaderTemplateProperty ()
-	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Internals.TemplatedItemsList`2<Microsoft.Maui.Controls.MultiPage`1<Microsoft.Maui.Controls.Page>, Microsoft.Maui.Controls.Page>:get_IsGroupingEnabledProperty ()
-	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Internals.TemplatedItemsList`2<Microsoft.Maui.Controls.MultiPage`1<Microsoft.Maui.Controls.Page>, Microsoft.Maui.Controls.Page>:get_ProgressiveLoadingProperty ()
-	Microsoft.Maui.Controls.BindablePropertyKey Microsoft.Maui.Controls.BindableProperty:CreateAttachedReadOnly (string,System.Type,System.Type,object,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.BindableProperty/ValidateValueDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangedDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangingDelegate,Microsoft.Maui.Controls.BindableProperty/CoerceValueDelegate,Microsoft.Maui.Controls.BindableProperty/CreateDefaultValueDelegate)
 	Microsoft.Maui.Controls.BindablePropertyKey Microsoft.Maui.Controls.BindableProperty:CreateReadOnly (string,System.Type,System.Type,object,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.BindableProperty/ValidateValueDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangedDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangingDelegate,Microsoft.Maui.Controls.BindableProperty/CoerceValueDelegate,Microsoft.Maui.Controls.BindableProperty/CreateDefaultValueDelegate)
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.AppThemeBinding:Clone ()
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Binding:Clone ()
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Xaml.AppThemeBindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Xaml.BindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.Brush:get_Default ()
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.Brush:op_Implicit (Microsoft.Maui.Graphics.Color)
-	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.NavigationPage:get_BarBackground ()
```

Comment:
Looks like the nav bar is not needed at all?

Reply:
Ok, I think the app is doing a web request (GET) that is timing out:

https://httpstat.us/200

This is not completing for me. Let me investigate and record it again.

Maybe we will be in OK shape if I fix this!

Reply:
Going to record again with: https://httpbin.org/status/200